src/elements/logos-block.html

```diff
@@ -0,0 +1,122 @@
+<link rel="import" href="../../bower_components/polymer/polymer-element.html">
```

Comment:
I think the appropriate name is PartnersBlock. LogosBlock is and old version, and always seemed to me weird